engine/schema/src/main/java/com/cloud/service/ServiceOfferingVO.java

```diff
@@ -110,7 +110,7 @@ public class ServiceOfferingVO implements ServiceOffering {
     private boolean defaultUse;
 
     @Column(name = "vm_type")
-    private String vmType;
+    private String systemVmType;
```

Comment:
Needed to change this variable otherwise we can't use it while building queries.

Reply:
@vishesh92 
can you explain ?
the change does not make sense.

Reply:
The field name should match the method name after `get` or `is`. The other option would be to change the method name. Otherwise the method interception fails.

Ref. code:
https://github.com/apache/cloudstack/blob/de095ba70d2a2261ffb2db7f3b8f29544939028e/framework/db/src/main/java/com/cloud/utils/db/SearchBase.java#L477-L503

Reply:
@vishesh92 
yes, renaming the method "getSystemVmType" to "getVmType" makes sense.

Reply:
agree, I would go the other way around (as @weizhouapache says)